docs/integrations/google_colab.rst

```diff
@@ -3,38 +3,108 @@
 Neptune-Google Colab Integration
 ================================
 
-You can run experiments on Google Colab and track them with Neptune.
+..
+    Video [1min screencast]
 
-Follow these steps:
+`Google Colab <https://colab.research.google.com/>`_ is a temporary runtime environment. This means you lose all your data (unless saved externally) once you restart your kernel. This is where you can leverage Neptune. By running your experiments on Google Colab and tracking them with Neptune, you can refer to the experiment stats and attributes, even after the Colab kernel has died.
```

Comment:
Please add links via |Link name| method. 

by doing that we can easily add target="_blank" to open those in new tabs.

Reply:
Could you please add an empty line after "... your kernel". 
Also, please add bullets for values that Neptune gives you.
Speaking of, I think it would be better to be explicit what you can log to Neptune from colab:

- parameters
- metrics and losses
- hardware consumption
- model checkpoints and other artifacts